diff --git a/index.js b/index.js
index ad05908..0ec460f 100644
--- a/index.js
+++ b/index.js
@@ -192,7 +192,7 @@ function createPublishPackageJson(packageJson, options) {
   if (rootMetadata.bugs !== void 0) {
     publishPackageJson.bugs = rootMetadata.bugs;
   }
-  if (rootMetadata.engines !== void 0) {
+  if (rootMetadata.engines !== void 0 && publishPackageJson.engines === void 0) {
     publishPackageJson.engines = rootMetadata.engines;
   }
   if (rootMetadata.license !== void 0) {
